modules/adb-lakehouse/azure_data_factory.tf

```diff
@@ -3,6 +3,6 @@ resource "azurerm_data_factory" "adf" {
 
   name                = var.data_factory_name
   location            = local.rg_location
-  resource_group_name = local.rg_name
+  resource_group_name = var.location
```

Comment:
it should be rg_name

Reply:
Fixed, I set local.rg_name again. Sorry, a typo there